### DISCUSSION TITLE
zabezpieczenie-routera-openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#121] [General]
# zabezpieczenie-routera-openwrt/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/zabezpieczenie-routera-openwrt/

**Comment by futrolab** [comment]
w "Panel administracyjny (https)" brakuje kroku System -> Administration -> HTTP(S) Access -> Ticked

**Reply by idarek** [reply]
A zainstalowałeś pakiet `luci-ssl` i `luci-app-uhttpd` tak jak opisano?
